src/backend/backend.ts

```diff
@@ -106,8 +127,34 @@ const updateComponentTree = (roots: Array<any>) => {
   previousCount = count;
 };
 
-const updateRouterTree = (routes: Array<MainRoute>) => {
-  messageBuffer.enqueue(MessageFactory.routerTree(routes));
+const updateLazyLoadedNgModules = (routers) => {
```

Comment:
Called on router change, this function parses any newly loaded NgModules from the routers and updates the frontend.